worldedit-bukkit/adapters/adapter-1_21_6/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_21_6/PaperweightPlatformAdapter.java

```diff
@@ -376,8 +391,19 @@ public static void sendChunk(IntPair pair, ServerLevel nmsWorld, int chunkX, int
                             nmsWorld.getLightEngine(),
                             null,
                             null,
-                            false // last false is to not bother with x-ray
+                            false // false so we can handle anti-X-Ray ourselves
```

Comment:
what happens if we just set this to `obfuscateAntiXRay`?

Reply:
We already don't guarantee chunks being sent before close returns